python/manylinux201x/build_arrow.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+  #!/bin/bash
```

Comment:
```suggestion
#!/bin/bash
```
This looks like an accident

Reply:
It is, thanks for spotting that!